### PR TITLE
Update 588 mission with final content DAH-734

### PR DESCRIPTION
**Pull request by akegan**
DAH-734

Small change based on feedback from persis

This PR:
- Updates the name of the preference to use "or"
- Changes the wording of the preference description in the application
- Updates the individual preference question to make it clearer that you just need to be an employee

### CODE REVIEW

**Review by james-wills-sf — APPROVED** [review]
@akegan lgtm

File: app/assets/javascripts/short-form/components/preferenceComponent.js.coffee [hunk 1]
```diff
@@ -52,7 +52,7 @@ angular.module('dahlia.components')
 
       # For 588 Mission preference, we need to override some defaults.
       @isEmploymentDisability = =>
-        @title == 'Employment/Disability Preference'
```
Comment (akegan) [inline app/assets/javascripts/short-form/components/preferenceComponent.js.coffee:55]: This is a bit fragile. We updated the name in salesforce and the extra field just disappeared 😕 . Hopefully won't happen again

File: app/assets/javascripts/short-form/components/preferenceComponent.js.coffee [hunk 2]
```diff
@@ -52,7 +52,7 @@ angular.module('dahlia.components')
...
+        @title == 'Employment or Disability Preference'
 
       @descriptionToTranslate = =>
         if @isEmploymentDisability() then "e7b_custom_preferences.employment_disability.description" else @translatedDescription
```
Comment (akegan) [inline app/assets/javascripts/short-form/components/preferenceComponent.js.coffee:58]: this isn't being recognized by grunt when looking for translations. need to use translate.instant or translate() or something to flag it so it isn't deleted

Comment (akegan) [reply app/assets/javascripts/short-form/components/preferenceComponent.js.coffee:58]: I fixed this in #1456

**Comment by akegan** [comment]
This is ready for review again. Hopefully can snag a final QA from @emilyshore before you go on leave if you're around tomorrow

**Review by james-wills-sf — APPROVED** [review]
@akegan lgtm

**Pull request merged**